iceberg/iceberg-handler/src/test/queries/positive/delete_iceberg_copy_on_write_partitioned.q

```diff
@@ -1,3 +1,4 @@
+--! qt:replace:/DeleteMetadataSpec(\S*)/#Masked#/
```

Comment:
why is this change?

Reply:
For masking the object ID since that was being printed in the explain plan.